src/RemoteResourceGitController.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2021 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const request = require('request-promise-native');
+const merge = require('deepmerge');
+const loggerFactory = require('./bunyan-api');
+const { BaseDownloadController } = require('@razee/razeedeploy-core');
+
+
+module.exports = class RemoteResourceGitController extends BaseDownloadController {
```

Comment:
Is there anything unique about this git controller other than its logger?

Reply:
I uploaded that as a template for @kdai7 so no it's pretty empty right now